docs/source/architecture/slashing.rst

```diff
@@ -21,12 +19,16 @@ TBD (https://github.com/nucypher/nucypher/issues/803)
 How slashing affects stake
 --------------------------
 
+.. _`Sub-stakes`: https://docs.nucypher.com/en/latest/architecture/sub_stakes.html
+
+The goal of slashing is to reduce the number of tokens that belongs to a staking offender. While the reduction of tokens is intended to be punitive, once the tokens have been slashed, the slashing algorithm attempts to preserve the most efficient use of the offenders' remaining tokens based on their existing `Sub-stakes`_. The main task is not to violate the logic of locking tokens.
+
 An entire stake consists of:
 
     * unlocked tokens which the staker can withdraw at any moment
     * tokens locked for a specific period
 
-In terms of how the stake is slashed, unlocked tokens are the first portion of the stake to be slashed. After that, if necessary, locked sub-stakes are decreased in order based on their remaining lock time, beginning with the shortest. The shortest sub-stake is decreased, and if the adjustment of that sub-stake is insufficient to fulfil the required punishment sum, then the next shortest sub-stake is decreased, and so on. The sub-stakes for each period are checked separately.
+In terms of how the stake is slashed, unlocked tokens are the first portion of the stake to be slashed. After that, if necessary, locked sub-stakes are decreased in order based on their remaining lock time, beginning with the shortest. The shortest sub-stake is decreased, and if the adjustment of that sub-stake is insufficient to fulfil the required punishment sum, then the next shortest sub-stake is decreased, and so on. Sub-stakes that begin in the next period are checked separately. Only the sub-stakes for the current and next periods need to be checked for slashing since sub-stakes can only be locked starting in the current or next period.
```

Comment:
One last thing (other parts are great!)
`Only the sub-stakes for the current and next periods need to be checked for slashing since sub-stakes can only be locked starting in the current or next period.` - slightly incorrect because starting period of sub-stake could be in a past.
Reason for changing that we don't care about past periods, current and next periods will be enough

Reply:
✔️